eth-connector/src/proof.rs

```diff
@@ -25,10 +25,42 @@ impl Proof {
                 .map_err(|_| ERR_BORSH_SERIALIZE)
                 .sdk_unwrap(),
         );
-        data.extend(self.header_data.clone());
+        data.extend_from_slice(&self.header_data);
         near_sdk::env::sha256(&data[..])
```

Comment:
And square brackets are redundant here, I guess
```suggestion
        near_sdk::env::sha256(&data)
```

Reply:
Removed in https://github.com/aurora-is-near/aurora-eth-connector/pull/18/commits/6ea9da52645d96591856a55264f379b02edbbb8c